counterparty-core/counterpartycore/lib/script.py

```diff
@@ -179,7 +179,7 @@ def base58_check_decode_py(s, version):
 
 def base58_check_decode(s, version):
     try:
-        decoded = b58.b58_decode(s)
+        decoded = bytes(b58.b58_decode(s))
```

Comment:
Now that we are relying on pyo3's [mapping of rust types to python types](https://pyo3.rs/v0.21.2/conversions/tables) we need to cast this list of bytes to bytes here.